plugins/processors/starlark/starlark.go

```diff
@@ -144,9 +89,9 @@ func (s *Starlark) Start(_ telegraf.Accumulator) error {
 }
 
 func (s *Starlark) Add(metric telegraf.Metric, acc telegraf.Accumulator) error {
-	s.args[0].(*Metric).Wrap(metric)
+	s.args[0].(*common.Metric).Wrap(metric)
```

Comment:
should this be a type check instead of a type assertion? if it fails it will panic. is that controlled by the user script?

Reply:
I simply did what we currently do in the [Starlark processor](https://github.com/influxdata/telegraf/blob/master/plugins/processors/starlark/starlark.go#L147). I can use a type check instead but if so I will modify the processor too for consistency. WDYT?